app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/pref/Prefs.kt

```diff
@@ -18,4 +19,21 @@ object Prefs : KotprefModel() {
             context.bool(R.bool.pref_default_value_enable_hide_bottom_navigation),
             R.string.pref_key_enable_hide_bottom_navigation
     )
+    var enableReopenPreviousRoomSessions: Boolean by booleanPref(
+            context.bool(R.bool.pref_default_value_enable_reopen_previous_room_sessions),
+            R.string.pref_key_enable_reopen_previous_room_sessions
+    )
+    var previousSessionTab: String by stringPref(
+            context.getString(R.string.pref_default_value_previous_session_tab)
+    )
+    var previousSessionTabId: Int by intPref(
+            context.integer(R.integer.pref_default_value_previous_session_tab_id)
+    )
+}
+
+fun initPreviousSessionPrefs() {
```

Comment:
I think this function move into Prefs. 👀 
What do you think?

Reply:
There is no against. 
I will move into Prefs.